ui/components/UploadImport.js

```diff
@@ -51,21 +51,29 @@ const styles = makeStyles(() => ({
 
 }));
 
-
-
-const UploadImport = ({ handleUpload, handleImport, configuration }) => {
+const UploadImport = ({ handleUpload, handleImport, configuration, isApplication, supportedTypes }) => {
   const classes = styles();
   const [open, setOpen] = React.useState(false);
   const [input, setInput] = React.useState();
   const [isError, setIsError] = React.useState(false);
+  const [fileType, setFileType] = React.useState();
+  const [sourceType, setSourceType] = React.useState();
 
-
+  const handleFilteType = (index) => {
```

Comment:
best to use a proper name for the function

Reply:
Thank you, fixed.